addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -127,7 +127,7 @@
     </t>
 
     <t t-name="web.embeddedActionsDropdown">
-        <Dropdown menuClass="'o_embedded_actions_dropdown_menu'">
+        <Dropdown menuClass="'o_embedded_actions_dropdown_menu'" t-if="!env.isSmall">
```

Comment:
If we add the condition here, the action will also be hidden in the app on small devices. Therefore, it should be hidden only in the project app